dev/sparktestsupport/modules.py

```diff
@@ -19,10 +19,48 @@
 import itertools
 import os
 import re
+import unittest
+
+from sparktestsupport import SPARK_HOME
 
 all_modules = []
 
 
+def _discover_python_unittests(paths):
+    """
+    Discover the python module which contains unittests under paths.
+
+    Such as:
+    ['pyspark/tests'], it will return the set of module name under the path of pyspark/tests, like
+    {'pyspark.tests.test_appsubmit', 'pyspark.tests.test_broadcast', ...}
+
+    :param paths: paths of module to be discovered.
+    :return: A set of complete test module name discovered udner the paths
```

Comment:
We should use numpydoc string style ... Feel free to make another PR to fix other docstrings later ..